Jellyfin.Api/Auth/FirstTimeSetupPolicy/FirstTimeSetupHandler.cs

```diff
@@ -24,24 +25,31 @@ public FirstTimeSetupHandler(IConfigurationManager configurationManager)
         /// <inheritdoc />
         protected override Task HandleRequirementAsync(AuthorizationHandlerContext context, FirstTimeSetupRequirement requirement)
         {
+            // Succeed if the startup wizard / first time setup is not complete
             if (!_configurationManager.CommonConfiguration.IsStartupWizardCompleted)
             {
                 context.Succeed(requirement);
             }
-            else if (requirement.RequireAdmin && !context.User.IsInRole(UserRoles.Administrator))
+
+            // Succeed if user is admin
+            else if (context.User.IsInRole(UserRoles.Administrator))
             {
-                context.Fail();
+                context.Succeed(requirement);
             }
-            else if (!requirement.RequireAdmin && context.User.IsInRole(UserRoles.Guest))
+
+            // Fail if admin is required and user is not admin
+            else if (requirement.RequireAdmin)
             {
                 context.Fail();
             }
-            else
+
+            // Succeed if admin is not required and user is not guest
```

Comment:
I don't think that's right. This is basically the authz handler for the setup "wizard". It should only allow first time setup and admins

Reply:
It has not worked in 10.9. Any of the endpoints using the FirstTimeSetupHandler where RequireAdmin is false are have been public for every 10.9 release. :melting_face: